lib/Driver/ToolChains.cpp

```diff
@@ -946,6 +960,21 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
                      /*AddRPath*/ true, TC);
 }
 
+static void
+addLinkSanitizerLibArgsForLinux(const ArgList &Args,
+                                 ArgStringList &Arguments,
+                                 StringRef Sanitizer, const ToolChain &TC) {
+
+  addLinkRuntimeLibForLinux(Args, Arguments,
+                    (Twine("libclang_rt.") + Sanitizer + "-" +
+                     "x86_64.a").str(), TC);
```

Comment:
OK. Let me figure out how to get the current architecture at that point, and I'll remove the hardcoding.